assignment-client/src/entities/EntityTreeSendThread.h

```diff
@@ -40,6 +42,7 @@ class EntityTreeSendThread : public OctreeSendThread {
 
     DiffTraversal _traversal;
     EntityPriorityQueue _sendQueue;
+    std::unordered_set<EntityItemPointer> _entitiesToSend;
```

Comment:
Not a blocker.  I want to get this logic in place, but we'll have to come back and fix it later.